freqtrade/tests/edge/test_edge_cli.py

```diff
@@ -0,0 +1,140 @@
+# pragma pylint: disable=missing-docstring, C0103, C0330
```

Comment:
positioning of this file is inconsistent - if the edge_cli file is in optimize, maybe the tests should be there too?